TESTING.asciidoc

```diff
@@ -291,6 +292,20 @@ A specific test case can be run with
   -Dtests.method="test {p0=cat.shards/10_basic/Help}"
 ---------------------------------------------------------------------------
 
+To run REST tests of licensed features, use the following command:
+
+---------------------------------------------------------------------------
+./gradlew ':x-pack:plugin:integTestRunner' \
+  --tests "org.elasticsearch.xpack.test.rest.XPackRestIT"
```

Comment:
Why specifying the tests direclty instead of running `:x-pack:plugin:integTest`?

Reply:
Agreed, there's only a single test suite for that project, I'd simplify the execution here.